drivers/lora/sx1276.c

```diff
@@ -38,6 +38,16 @@ LOG_MODULE_REGISTER(sx1276);
 #define GPIO_PA_BOOST_ENABLE_FLAGS			\
 	DT_INST_GPIO_FLAGS(0, pa_boost_enable_gpios)
 
+#define GPIO_TCXO_POWER_PIN	DT_INST_GPIO_PIN(0, tcxo_power_gpios)
+#define GPIO_TCXO_POWER_FLAGS	DT_INST_GPIO_FLAGS(0, tcxo_power_gpios)
+
+#if DT_INST_NODE_HAS_PROP(0, tcxo_power_startup_delay_ms)
```

Comment:
Just for my understanding. From where this delay requirement comes in? Board design or crystal osc?

Reply:
A TCXO (TC stands for Temperature Controlled) contains a tiny oven that keeps the crystal at a well-defined temperature. Due to the physics of the device, it'll take some time between turning on power and the device reaching its target temperature. The frequency of the crystal is only well defined once it reaches its target temperature.

The trick here is that you want the delay to be enough to heat the crystal, but not too long since that wastes battery.